bh_modules/pascalkeywords.py

```diff
@@ -0,0 +1,14 @@
+"""
+BracketHighlighter.
+
+Copyright (c) 2013 - 2015 Isaac Muse <isaacmuse@gmail.com>
+License: MIT
+"""
+
+
+def compare(name, first, second, bfr):
+    """Differentiate 'repeat..until' and 'unit..end.' from '*..end' brackets."""
+    brackets = (bfr[first.begin:first.end], bfr[second.begin:second.end])
+    return (brackets[0] == "repeat" and brackets[1] == "until") \
+        or (brackets[0] == "unit" and brackets[1] == "end.") \
```

Comment:
Is `(brackets[0] == "unit" and brackets[1] == "end.")` and `(brackets[0] != "repeat" and brackets[0] != "unit" and (brackets[1] == "end"))` necessary?  I feel like you could remove both and just have: `(brackets[0] != "repeat" and (brackets[1] == "end"))`.  From the code, it appears that `until` is only used for `repeat` and all others use `end`, or am I misinterpreting?